src/Adapter.php

```diff
@@ -6,17 +6,24 @@
 
 use InvalidArgumentException;
 use Yiisoft\Queue\Adapter\AdapterInterface;
+use Yiisoft\Queue\Cli\LoopInterface;
 use Yiisoft\Queue\Enum\JobStatus;
 use Yiisoft\Queue\Message\MessageInterface;
 use Yiisoft\Queue\QueueFactory;
 use Yiisoft\Queue\Message\IdEnvelope;
 use Yiisoft\Db\Connection\ConnectionInterface;
 use Yiisoft\Db\Query\Query;
+use Yiisoft\Mutex\MutexFactoryInterface;
+use Yiisoft\Mutex\MutexInterface;
 
 final class Adapter implements AdapterInterface
 {
     /**
-     * @var int timeout
+     * @var MutexInterface mutex interface
```

Comment:
We're usually starting descriptions with capital letter and end these with dot. Not a huge issue though. Can fix it later.

```suggestion
     * @var MutexInterface Mutex interface.
```